### PR TITLE
fix os.getcwd() FileNotFoundError

### PR DESCRIPTION
**Pull request by mortimer2015**
fix os.getcwd() FileNotFoundError

### CODE REVIEW

File: celery/utils/imports.py [hunk 1]
```diff
@@ -51,8 +51,13 @@ def instantiate(name, *args, **kwargs):
 @contextmanager
 def cwd_in_path():
     """Context adding the current working directory to sys.path."""
-    cwd = os.getcwd()
-    if cwd in sys.path:
+    try:
```
Comment (auvipy) [inline celery/utils/imports.py:54]: could you please add some unit test to verify the change?

**Review by auvipy — CHANGES_REQUESTED** [review]

File: celery/utils/imports.py (hunk 1 above)
Comment (mortimer2015) [reply celery/utils/imports.py:54]: Unit test added

**Comment by Nusnus** [comment]
@mortimer2015 hey - please notice the tests are failing :)

**Comment by Nusnus** [comment]
It still does unfortunately @mortimer2015 
Make sure you're also rebased on the latest `main`

**Comment by Nusnus** [comment]
Seems to be partially fixed. Only Windows tests are failing now. @mortimer2015

**Comment by auvipy** [comment]
After checking the pypy failures it seems to be a network issue of github. so I triggered one job to recheck. but could not re run two at once as github do not allow it now

**Review by auvipy — APPROVED** [review]

**Comment by auvipy** [comment]
@Nusnus if you have any inputs please share when you have some time.

**Review by Nusnus — APPROVED** [review]

**Pull request merged**